docs/guides/schema-export.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Schema export
+---
+
+# Schema export
+
+Sometimes IDE plugins and code generation tools require you to provide a GraphQL schema
+definition.
+
+Strawberry provides a command to export your schema definition.
+The exported schema will be described in the GraphQL schema definition language (SDL).
+
+You can export your schema using the following command:
+
+    strawberry export_schema app
```

Comment:
````suggestion
```sh
strawberry export_schema app
```
````

Reply:
The "Builtin server" guide used indentation to create a red box around the server command. Thats why I choose indentation over a code snippet. Maybe both could be combined?

Reply:
Oh right, I forgot that indentation is a code block in markdown. It's probably fine to leave as-is